addons/mail/static/src/core/common/message_model.js

```diff
@@ -126,15 +126,14 @@ export class Message extends Record {
     is_transient;
     linkPreviews = Record.many("LinkPreview", { inverse: "message", onDelete: (r) => r.delete() });
     /** @type {number[]} */
-    needaction_partner_ids = [];
     parentMessage = Record.one("Message");
     reactions = Record.many("MessageReactions", { inverse: "message" });
     notifications = Record.many("Notification", { inverse: "message" });
     recipients = Record.many("Persona");
     thread = Record.one("Thread");
     threadAsNeedaction = Record.one("Thread", {
```

Comment:
Unrelated to PR but inverse (`needactionMessages`) has a `sort` but we never use anything else than its length now, so it could be simplified.

But why do we even use that field? I think it should just be `message_needaction_counter` instead.

Reply:
It's used inside the messaging_menu, the sort is there to display the last need action message